v2/thirdpartyemailpassword/common-customizations/embed-sign-in-up-form.mdx

```diff
@@ -77,7 +290,29 @@ import Header from "./header";
 import Footer from "./footer";
 import { useNavigate } from "react-router-dom";
 
-function MyLandingPage() {
+SuperTokens.init({
+    appInfo: {
+        apiDomain: "...",
+        appName: "...",
+        websiteDomain: "..."
+    },
+    recipeList: [
+        ThirdPartyEmailPassword.init({
+            // ...
+        }),
+        // ...
+    ],
+    // highlight-start
+    async getRedirectionURL(context) {
+        // The user will be taken to this path when they need to login.
+        if (context.action === "TO_AUTH") {
+            return "/auth"; // return the path where you are rendering the Auth UI
+        };
+    },
```

Comment:
No need to have this.

Reply:
done

Reply:
Im a little confused. You said it's done, but the snippet is still here.